src/NewTools-Debugger-Commands/StDebuggerCommand.class.st

```diff
@@ -34,6 +34,12 @@ StDebuggerCommand class >> forSpecContext: anObject [
 		yourself
 ]
 
+{ #category : #testing }
+StDebuggerCommand class >> isAbstract [
+
+	^ self == StDebuggerCommand
+]
+
```

Comment:
I find it odd to apply that everywhere: the debugger commands (and some in the same commit, like Sindarin) are not meant to be tested as "abstract".

Is that rule for abstract classes we have to apply everywhere if Pharo?
Is it not a bit heavy?
And what would be the usage for sindarin and stdebugger commands?
Except that which bothers me (but if that is normal, please dismiss my comment), the PR is simple and should be merged.

Reply:
Please read:
https://github.com/pharo-project/pharo/issues/9211